src/reanimated2/Colors.ts

```diff
@@ -31,50 +36,39 @@ function call(...args: unknown[]): string {
   return '\\(\\s*(' + args.join(')\\s*,\\s*(') + ')\\s*\\)';
 }
 
-// matchers use RegExp objects which needs to be created separately on JS and on
-// the UI thread. We keep separate cache of Regexes for UI and JS using the below
-// objects, then pick the right cache in getMatchers() method.
-const jsCachedMatchers: Matchers = {};
-const uiCachedMatchers: Matchers = !isConfigured() ? {} : makeRemote({});
 type Matchers = {
-  rgb?: RegExp;
-  rgba?: RegExp;
-  hsl?: RegExp;
-  hsla?: RegExp;
-  hex3?: RegExp;
-  hex4?: RegExp;
-  hex5?: RegExp;
```

Comment:
@tjzel noticed that `hex5` is gone. Can you please make sure it doesn't break anything?